### PR TITLE
[14.0][MIG] hr_attendance_modification_tracking

### PR DESCRIPTION
**Pull request by aphon61bank**
I'm edit track_visibility = tracking.

### CODE REVIEW

File: hr_attendance_modification_tracking/tests/test_hr_attendance.py [hunk 1]
```diff
@@ -83,7 +83,9 @@ def test_attendance_edit(self):
         dto = datetime.now() + relativedelta(hours=1)
         att.write({"check_out": dto.strftime(DF)})
         self.assertEqual(
-            att.time_changed_manually, True, "Use case 5: Wrong value, data changed."
```
Comment (ao-landoo) [inline hr_attendance_modification_tracking/tests/test_hr_attendance.py]: I don't think changing this assert from True to False is something valid. This shows the behavour has been changed in the migration.

File: hr_attendance_modification_tracking/tests/test_hr_attendance.py [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        )
+        # Use case 5:
+        # Change previous attendance check-out to now + 1 hour
+        # Expected: manually_changed = True
+        dto = datetime.now() + relativedelta(hours=1)
+        att.write({"check_out": dto.strftime(DF)})
+        self.assertEqual(
+            att.time_changed_manually,
+            False,
+            "Use case 5: Wrong value, data changed.",
```
Comment (ao-landoo) [inline hr_attendance_modification_tracking/tests/test_hr_attendance.py:88]: I don't think changing this assert from True to False is something valid. This shows the behavour has been changed in the migration.

Comment (ao-landoo) [reply hr_attendance_modification_tracking/tests/test_hr_attendance.py:88]: What do you think @Saran440 ??

**Comment by Saran440** [comment]
Close, Move on https://github.com/OCA/hr-attendance/pull/36

**Pull request closed without merging**